cmacro_import removes subsequent token

I have a file like this:

```c
#include <stdio.h>

cmacro_import "magma/util/doto.c"

int main(int argc, char **argv) {
  doto("hello %s") {
    printf("world!\n");
  }
  return 0;
}
```
When I compile it, gcc gives me this warning:
```
main.c:2:4: warning: return type defaults to ‘int’ [-Wimplicit-int]
    main (int argc, char** argv) {
```
Sure enough, the generated C looks like this:
```c
#include <stdio.h>
   main (int argc, char** argv) {
printf ("hello %s", "world!\n"); return 0;
}
```
It looks like the `cmacro_import` call destroys the `int` before `main`. I tried with a statement like `char c = 'c';` after `cmacro_import` and saw the same thing; the generated C looked like `c = 'c';` and gcc errored.

Am I missing something or is this a bug? Thanks, this otherwise looks like a really awesome way to program C!

Software:

* cmacro 3b1400f24dc2ad9b295e8bbcf897d73dbe5f8ae4
* magma 7be0a77fe7b1f9d2f928bf27fb8b4423b02e3a8d
* SBCL 1.3.1.debian

--- a/src/parser.lisp
+++ b/src/parser.lisp
@@ -277,7 +277,7 @@
               (progn
                 (let ((path (second sub-ast)))
                   ;; Remove the macro
-                  (setf sub-ast (cddr sub-ast))
+                  (setf sub-ast (cdr sub-ast))
                   (if (typep path '<string>)
                       (parse-string
                        (slurp-file (parse-namestring (string-text

